Packages/com.unity.inputsystem/InputSystem/Controls/ButtonControl.cs

```diff
@@ -20,7 +18,8 @@ namespace UnityEngine.InputSystem.Controls
     public class ButtonControl : AxisControl
     {
         public float pressPoint;
-        public float pressPointOrDefault => pressPoint > 0.0f ? pressPoint : InputSystem.settings.defaultButtonPressPoint;
+
+        internal float pressPointOrDefault => pressPoint > 0.0f ? pressPoint : InputSystem.settings.defaultButtonPressPoint;
```

Comment:
was this what we decided on? Didn't we end up concluding that press points were the more desirable solution after all, as they might be device/control specific? (Not sure if that means they should be public or not, just want to make sure this is what we want).

Reply:
Yup, no change in agreement on press points and no change in behavior. Merely removed this particular form of getter from the public API. But looking at it, maybe better the other way round.

Reply:
what do we do with this? Don't care strongly, tbh.

Reply:
Heh, me neither :) Fine either way. I'll go revert the change here. Might actually be a useful one.